src/GovUk.Education.ExploreEducationStatistics.Common/GovUk.Education.ExploreEducationStatistics.Common.csproj

```diff
@@ -1,37 +1,34 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
-        <LangVersion>10.0</LangVersion>
+        <TargetFramework>net8.0</TargetFramework>
         <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     </PropertyGroup>
 
-
     <ItemGroup>
         <CompilerVisibleProperty Include="RootNamespace" />
         <CompilerVisibleProperty Include="ProjectDir" />
     </ItemGroup>
-  
+
     <ItemGroup>
+        <FrameworkReference Include="Microsoft.AspNetCore.App" />
         <PackageReference Include="AngleSharp" Version="1.0.4" />
         <PackageReference Include="AspectInjector" Version="2.8.2" PrivateAssets="all" />
         <PackageReference Include="Aspects.Universal" Version="1.0.1" />
-        <PackageReference Include="AutoMapper" Version="9.0.0" />
+        <PackageReference Include="AutoMapper" Version="13.0.1" />
         <PackageReference Include="Azure.Storage.Blobs" Version="12.17.0" />
-        <PackageReference Include="CsvHelper" Version="30.0.1" />
+        <PackageReference Include="CsvHelper" Version="31.0.2" />
         <PackageReference Include="FluentValidation" Version="11.9.0" />
         <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="11.9.0" />
-        <PackageReference Include="IsExternalInit" Version="1.0.3" PrivateAssets="all" />
-        <PackageReference Include="Microsoft.ApplicationInsights" Version="2.21.0" />
-        <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
-        <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="6.0.21" />
+        <PackageReference Include="Microsoft.ApplicationInsights" Version="2.22.0" />
+        <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="8.0.3" />
         <!-- TODO EES-4202 DataMovement depends on deprecated Microsoft.Azure.Storage.Blob SDK v11 -->
         <PackageReference Include="Microsoft.Azure.Storage.DataMovement" Version="2.0.4" />
-        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.21" />
-        <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="6.0.21" />
-        <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.21" />
-        <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0" />
-        <PackageReference Include="Namotion.Reflection" Version="3.0.0" />
+        <PackageReference Include="Microsoft.Data.SqlClient" Version="5.2.0" />
```

Comment:
This is new?

Reply:
It's new to our config but it's always been included in our build because it's a dependency of `Microsoft.EntityFrameworkCore.SqlServer`.

`Microsoft.EntityFrameworkCore.SqlServer` has a dependency on `Microsoft.Data.SqlClient >= 5.1.5` but `5.1.5` causes an error on startup of projects with .NET 8.

```
[admin] System.TypeLoadException: Could not load type 'SqlGuidCaster' from assembly 'Microsoft.Data.SqlClient, Version=5.0.0.0, Culture=neutral, PublicKeyToken=23ec7fc2d6eaa4a5' because it contains an object field at offset 0 that is incorrectly aligned or overlapped by a non-object field.
```

There's a https://github.com/dotnet/SqlClient issue about this problem and version `5.2.0` is the fix.